tests/foreman/ui/test_errata.py

```diff
@@ -395,88 +481,239 @@ def test_end_to_end(
+        14. Assert no pagination on new host UI>content>errata.
+        15. Raise `NoSuchElementException` when looking for a pagination element.
+
+    :expectedresults: More than just the current page of errata can be selected
+        and applied, with changed per-page settings.
 
     :customerscenario: true
 
     :BZ: 1662254, 1846670
     """
-
-    org = function_org_with_parameter
-    pkgs = ' '.join(FAKE_3_YUM_OUTDATED_PACKAGES)
-    repos_collection = target_sat.cli_factory.RepositoryCollection(
-        distro='rhel7',
-        repositories=[
-            target_sat.cli_factory.SatelliteToolsRepository(),
-            target_sat.cli_factory.YumRepository(url=settings.repos.yum_3.url),
-        ],
+    org = module_sca_manifest_org
+    all_repos = module_target_sat.api.Repository(organization=org).search()
+    # custom_repo was created & added to cv, in registered_contenthost fixture
+    # search for the instance
+    custom_repo = [repo for repo in all_repos if repo.url == CUSTOM_REPO_3_URL]
+    assert len(custom_repo) == 1
+    custom_repo = custom_repo[0].read()
+    custom_repo.sync()
+    # Set up rh_repo
+    rh_repo_id = module_target_sat.api_factory.enable_rhrepo_and_fetchid(
+        basearch=DEFAULT_ARCHITECTURE,
+        org_id=org.id,
+        product=PRDS['rhel8'],
+        repo=REPOS['rhst8']['name'],
+        reposet=REPOSET['rhst8'],
+        releasever='None',
     )
-    repos_collection.setup_content(org.id, lce.id)
-    with Broker(nick=repos_collection.distro, host_class=ContentHost) as client:
-        client.add_rex_key(satellite=target_sat)
-        # Add repo and install packages that need errata
-        repos_collection.setup_virtual_machine(client)
-        assert client.execute(f'yum install -y {pkgs}').status == 0
-        with session:
-            # Go to content host's Errata tab and read the page's pagination widgets
-            session.organization.select(org_name=org.name)
-            session.location.select(loc_name=DEFAULT_LOC)
-            page_values = session.contenthost.read(
-                client.hostname, widget_names=['errata.pagination']
-            )
-            assert int(page_values['errata']['pagination']['per_page']) == 20
-            # assert total_pages > 1 with default page settings
-            assert int(page_values['errata']['pagination']['pages']) > 1
+    rh_repo = module_target_sat.api.Repository(id=rh_repo_id).read()
+    rh_repo.sync()
+    # add rh_repo to cv, publish version and promote w/ the repository
+    module_target_sat.cli.ContentView.add_repository(
+        {
+            'id': module_cv.id,
+            'organization-id': org.id,
+            'repository-id': rh_repo_id,
+        }
+    )
+    module_cv = module_cv.read()
+    cv_publish_promote(
+        module_target_sat,
+        org,
+        module_cv,
+        module_lce,
+    )
+    registered_contenthost.add_rex_key(satellite=module_target_sat)
+    assert registered_contenthost.subscribed
```

Comment:
This is not require, as subscription status already asserted inside `registered_contenthost` fixture after registration of content host.
```suggestion
```